src/renderers/dom/shared/inputValueTracking.js

```diff
@@ -144,9 +143,13 @@ var inputValueTracking = {
     }
 
     var lastValue = tracker.getValue();
-    var nextValue = getValueFromNode(
-      ReactDOMComponentTree.getNodeFromInstance(inst),
-    );
+
+    var node = inst;
+    if (!(inst: any).nodeName) {
```

Comment:
yeah its not great, I'm trying to detect if `inst` is a component instance, like it is in stack, or a DOM node as in fiber. This is the most minimal DRY change I could think of. Alternatively we could split this method into two, but its a bunch of duplication.